wg-data-protection/annual-report-2021.md

```diff
@@ -1,62 +1,76 @@
 # 2021 Annual Report: WG Data Protection
 
+Note: Only include KEPs targeting 1.21, 1.22, and 1.23 releases.
+
 ## Current initiatives
 
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+- [Data Protection White Paper](https://github.com/kubernetes/community/blob/master/wg-data-protection/data-protection-workflows-white-paper.md)
+- KEPs being worked on:
+  - Restrict volume mode conversion to fix security issue: Had discussions in the WG and submitted KEP to introduce it as an Alpha feature: https://github.com/kubernetes/enhancements/pull/3151
+  - Working on moving Volume Populator to Beta: https://github.com/kubernetes/enhancements/pull/2934
+  - Object Storage API (COSI): KEP is being updated and reviewed: https://github.com/kubernetes/enhancements/pull/2813
+  - ContainerNotifier KEP review in progress: https://github.com/kubernetes/enhancements/pull/1995/
+  - Volume Group KEP review in progress: https://github.com/kubernetes/enhancements/pull/1551
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+We discussed the following topics in the WG that are not tracked in KEPs:
+- Change Block Tracking (CBT) API design: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.1olwavha9frv
+- Volume Replication: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.pah3yke9ddug
+- Backup and restore externally managed services:
+  - https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.7zwanijz69u1
+  - https://docs.google.com/presentation/d/1IM6d0w3CDdHv1dLaFNXEcxy5fuDTr9LERAdMVkZiK9s/edit#slide=id.p
+- Snapshot policy (immutable snapshot): https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.gb8m7t8jro1v
+- Securing S3 Backups against Ransomware: https://sched.co/igUT
+- Volume Snapshot GA phases: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.w8v8tpkuw8ac
+- Kubernetes Data Protection with Velero: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.iekhl8nl58lo
 
 ## Project health
 
 1. What's the current roadmap until completion of the working group?
 
-   -
-   -
-   -
+We have identified the missing building blocks for supporting Data Protection in Kubernetes in our white paper: https://github.com/kubernetes/community/blob/master/wg-data-protection/data-protection-workflows-white-paper.md#what-are-the-missing-building-blocks-in-kubernetes.  We will continue to work on them until all the missing pieces are available in Kubernetes.
 
 2. Does the group have contributors from multiple companies/affiliations?
 
-   -
+Yes
```

Comment:
can you list a few of the top contributing orgs here?

Reply:
Based on the meeting minutes log here: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#

Those are the companies currently listed as having shown interest in this WG: Arrikto, Catalogic Software, Cohesity, Commvault, Dell EMC, Druva, Google, HPE, IBM, Infinidat, Kasten by Veeam, LINBIT, Linkedin, MayaData, Microsoft, Mongo, NetApp, Pure Storage (PortWorx), Red Hat, Rubrik, SUSE, Trilio, Veritas, VMware

Reply:
Referenced the supporting companies in the agenda doc. Also listed companies actively participating in the WG.